snippets/desktop-guide/wpf/styles-templates-create-apply-template/CreateTemplate.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Do you really need to check the solution file in?

Reply:
Visual Studio hates it when you don't have SLN files and always wants you to create one. So it's just easier to keep one and not be bothered by it.

Reply:
We specifically have a rule about not checking in solution files when there's only one project:
5. Don't check in the solution file if it contains only one project.

https://github.com/dotnet/samples/blob/master/README.md

Reply:
Good to know. We had the same rule in Visual Studio, because you don't need the solution file to compile the project.

Reply:
Good to know. :)